docs/frontend/content-elements.md

```diff
@@ -4,10 +4,21 @@
 
 📍 Each content element contain base and common props.
 
-📍 Each content element has `Ce` prefix ( We would like to avoid conflicts with other libraries or with your UI components )
+📍 Each content element has `Ce` prefix
```

Comment:
4 lines below recommended naming convention is `T3Ce`, so we have to consider if `T3Ce` is reserved for native typo3 content elements or for all content elements - even custom.

Reply:
for all content elements - default ones from TYPO3 and all custom which API deliver